iopipe/report.py

```diff
@@ -121,23 +120,19 @@ def send(self, error=None):
 
         self.report['environment']['host']['boot_id'] = system.read_bootid()
 
-        self.report['environment']['os']['linux']['mem'] = meminfo = system.read_meminfo()
+        meminfo = system.read_meminfo()
 
         self.report.update({
             'aws': self.extract_context_data(),
             'duration': int(duration * 1e9),
```

Comment:
@pselle @ewindisch Do we want to measure duration before or after system reads? Currently in the node.js agent it is done after system reads. But I'm not sure how useful that is. I'm thinking we would want to measure immediately after the function is invoked. The way the node.js agent currently does it isn't terribly useful since it doesn't take into account the time it takes to actually send the report. Also system read time can very greatly. So I think the user would be more interested in how long their function took to execute and not so much system reads / http requests.